cmd/brew-services.rb

```diff
@@ -236,6 +237,33 @@ def restart(target)
       end
     end
 
+    # "load" a plist
+    def launchctl_load(plist, _function, service)
```

Comment:
But it looks like it probably wants to be called `function` and used in `ohai("Successfully started `#{service.name}` (label: #{service.label})")`